awscli/autoprompt/factory.py

```diff
@@ -147,30 +151,36 @@ def create_layout(self, on_input_buffer_text_changed=None,
         # - A toolbar denoting key bindings.
         # - A help panel
         # - A debug panel in case debug mode enabled
-        if search_field is None:
-            search_field = SearchToolbar()
         if input_buffer_container is None:
             input_buffer = \
                 self.create_input_buffer(on_input_buffer_text_changed)
             input_buffer_container = \
                 self.create_input_buffer_container(input_buffer)
         if doc_window is None:
             doc_buffer = self.create_doc_buffer()
-            doc_window = self.create_doc_window(doc_buffer, search_field)
+            doc_window = self.create_bottom_window('Doc panel', doc_buffer)
+        if output_window is None:
+            output_buffer = self.create_output_buffer()
+            output_window = self.create_bottom_window(
+                'Output panel', output_buffer)
         return Layout(
             HSplit([
                 VSplit([
                     HSplit([
                         input_buffer_container,
-                        ConditionalContainer(HorizontalLine(),
-                                             doc_section_visible),
-                        ConditionalContainer(doc_window,
-                                             doc_section_visible),
+                        VSplit([
+                            ConditionalContainer(doc_window,
+                                                 doc_section_visible),
+                            ConditionalContainer(
+                                VerticalLine(),
+                                output_section_visible & doc_section_visible),
+                            ConditionalContainer(output_window,
+                                                 output_section_visible),
+                        ]),
```

Comment:
To clean up the top level view of the layout, it would be nice if we just represented this `VSplit` as a single variable. Maybe call it `bottom_window` and have it be returned from the `create_bottom_window` method? Then we can rename the current `create_bottom_window` method that creates the different types of windows to `create_searchable_window()`?